src/bika/lims/browser/analyses/view.py

```diff
@@ -519,6 +519,18 @@ def load_analysis_categories(self):
     def append_partition_filters(self):
         """Append additional review state filters for partitions
         """
+
+        # view is used for instrument QC view as well
+        if not IAnalysisRequest.providedBy(self.context):
+            return
+
+        # check if the sample has partitions
+        partitions = self.context.getDescendants()
+
+        # root partition w/o partitions
+        if not partitions:
+            return
```

Comment:
Thanks, I noticed yesterday the filter button by Sample ID was being displayed in analyses listing although the Sample did not have partitions!.